### PR TITLE
Add Tarun-Kamboj

### LINKED ISSUE #419: Tarun-Kamboj

### PR DESCRIPTION
**Pull request by Tarun-Kamboj**
Resolve #419

### CODE REVIEW

**Comment by vikkastiwari** [comment]
It seems great to me ;)

File: data.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 [
+  {
```
Comment (AllanRegush) [inline data.json:2]: Please add yourself to the bottom :smile:

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1: Just a minor adjustment. :nerd_face:

**Comment by Tarun-Kamboj** [comment]
@AllanRegush is it ok now?

**Review by galahad42 — APPROVED** [review]

**Review by schmelto — CHANGES_REQUESTED** [review]
Think what @AllanRegush ment was right at the end of the document (Line 697)

File: data.json [hunk 2]
```diff
@@ -83,6 +83,12 @@
     "imageUrl": "https://user-images.githubusercontent.com/63111742/88703846-eed9f000-d115-11ea-9569-587de740a27a.jpg",
     "issueNumber": 39
   },
+  {
```
Comment (AllanRegush) [inline data.json:86]: Please add yourself to the end of the `data.json1 file.

**Review by AllanRegush — CHANGES_REQUESTED** [review]
Looking Good :+1: Just a minor adjustment. :nerd_face:

**Comment by Tarun-Kamboj** [comment]
Requested changes done. @AllanRegush

**Review by schmelto — APPROVED** [review]

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you :smile:

**Pull request merged**